README.md

```diff
@@ -84,17 +84,17 @@ IEnumerable<MyDataType> listResult = await list.ReadAllAsync(CancellationToken.N
 // listResult contains [ MyDataType("list item 1"), MyDataType("list item 2") ]
 ```
 
-Full library documentation can be found at the [docs site](https://wayfair-incubator.github.io/AeroSharp/)
+Full library documentation can be found at the [docs site](https://wayfair-incubator.github.io/AeroSharp/).
 
-Code examples can be found in the [examples directory](./examples)
+Code examples can be found in the [examples directory](./examples).
```

Comment:
Do relative paths work in NuGet? 

Have you previewed the ReadMe per https://devblogs.microsoft.com/nuget/add-a-readme-to-your-nuget-package/#preview-your-readme ?

Reply:
Good call. I'll adjust the links in this README to be full paths.